digesta-bot/action.yml

```
@@ -65,7 +69,18 @@ runs:
                   continue
                 fi
                 echo "Processing ${images2[i]} in file $file"
-                updated_digest=$(crane digest "${images2[i]}")
+
+                crane digest "${images2[i]}" > digest.log 2> logerror.txt
+                if [ $? -eq 0 ]; then
+                    updated_digest=$(cat digest.log)
+                    rm -f digest.log
+                else
+                    echo "Failed to retrive digest info for ${images2[i]}"
+                    echo "Failed to retrive digest info for ${images2[i]}" >> "$GITHUB_STEP_SUMMARY"
+                    cat logerror.txt >> "$GITHUB_STEP_SUMMARY"
+                    rm -f logerror.txt
+                fi
```

move `rm -f logerror.txt` to after conditional to always cleanup.
```suggestion
                fi
                rm -f logerror.txt
                rm -f digest.log
```